airbyte-integrations/connectors/destination-redshift/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 airbyteJavaConnector {
     cdkVersionRequired = '0.10.1'
     features = ['db-destinations', 's3-destinations']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
Can we move this changes to a different branch. so dv_cdk_changes is CDK only

Reply:
I'd actually prefer to keep it in to trigger redshift integration tests against the updated cdk code - generally I think it's good practice to run at least one connector's CI for every CDK change